Integrations/GitHub/GitHub.py

```diff
@@ -0,0 +1,253 @@
+    LOG("GitHub: Fetching open issues for repository: {}/{}".format(OWNER, REPO))
+    context_entries = []
+    res = http_request('GET', "/issues")
+    for issue in res:
+        context_entries.append({
+            'ID': issue.get('number'),
+            'Title': issue.get('title'),
+            'Body': issue.get('body'),
+            'Assignees': [assignee.get('login') for assignee in issue.get('assignees')],
+            'Labels': [label.get('name') for label in issue.get('labels')]
+        })
+    HR = tableToMarkdown("Open issues", context_entries)
+    EC = {'GitHub.Issue(val.ID && val.ID == obj.ID)': context_entries}
+    return_outputs(HR, EC, res)
+
+
+def create_issue_command():
+    title = demisto.args().get('title')
+    body = demisto.args().get('body')
+    assignees = argToList(demisto.args().get('assignees'))
+    labels = argToList(demisto.args().get('labels'))
+
+    LOG("GitHub: Creating a new issue with title %s" % (title))
+
+    create_issue(title, body, assignees, labels)
+
+
+def create_issue(title, body, assignees, labels):
+    data = {'title': title}
+    if (body is not None):
+        data['body'] = body
+    if (len(assignees) > 0):
+        ##check assignees
+        data['assignees'] = assignees
+    if (len(labels) > 0):
+        data['labels'] = labels
+    res = http_request('POST', "/issues", data)
+
+    context_entries = {
+        'ID': res.get('number'),
+        'Title': res.get('title'),
+        'Body': res.get('body'),
+        'Assignees': [assignee.get('login') for assignee in res.get('assignees')],
+        'Labels': [label.get('name') for label in res.get('labels')]
+    }
+    HR = tableToMarkdown("Created issue", context_entries,
+                         ["ID", "Title", "Body", "Assignees", "Labels"])
+    EC = {'GitHub.Issue(val.ID && val.ID == obj.ID)': context_entries}
+    return_outputs(HR, EC, res)
+
+
+def close_issue_command():
+    issues = argToList(str(demisto.args().get('issue_number')))
+    s = ""
+    for issue in issues:
+        close_issue(issue)
+        s = s + str(issue) + ", "
+    demisto.results("Closed issues number: %s" % s[:-2])
+
+
+def close_issue(issue_number):
+    data = {"state": "closed"}
+    http_request("POST", "/issues/" + str(issue_number), data)
+
+
+def update_issue_command():
+    title = demisto.args().get('title')
+    body = demisto.args().get('body')
+    assignees = argToList(demisto.args().get('assignees'))
+    labels = argToList(demisto.args().get('labels'))
+    issue_number = str(demisto.args().get('issue_number'))
+
+    LOG("GitHub: Updating issue number {}".format(issue_number))
+
+    return (update_issue(issue_number, title, body, assignees, labels))
+
+
+def update_issue(issue_number, title, body, assignees, labels):
+    data = {}
+    if (title is not None):
+        data['title'] = title
+    if (body is not None):
+        data['body'] = body
+    if (len(assignees) > 0):
+        ##check assignees
```

Comment:
wrote it to myself and forgot to delete the comment. 
like close issue - this would return a 422 status which is explicitly taken care of in the http requests functoion